src/main/java/uk/gov/hmcts/reform/professionalapi/service/PaymentAccountService.java

```diff
@@ -1,12 +1,19 @@
 package uk.gov.hmcts.reform.professionalapi.service;
 
 import uk.gov.hmcts.reform.professionalapi.controller.request.PbaEditRequest;
+import uk.gov.hmcts.reform.professionalapi.controller.request.PbaRequest;
+import uk.gov.hmcts.reform.professionalapi.controller.response.UpdatePbaStatusResponse;
 import uk.gov.hmcts.reform.professionalapi.domain.Organisation;
 import uk.gov.hmcts.reform.professionalapi.domain.PbaResponse;
 
+import java.util.List;
+
 public interface PaymentAccountService {
 
     Organisation findPaymentAccountsByEmail(String email);
 
     PbaResponse editPaymentAccountsByOrganisation(Organisation organisation, PbaEditRequest pbaEditRequest);
+
+    UpdatePbaStatusResponse updatePaymentAccountsForAnOrganisation(List<PbaRequest> pbaRequestList, String orgId);

```

Comment:
please rename method updatePaymentAccountsForAnOrganisation to updatePaymentAccountsStatusForAnOrganisation

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/1096/commits/0d9a5fb64ad41cfae4999baffff40ff0a8622ce7